pkg/tar/tar.go

```diff
@@ -255,3 +275,13 @@ func extractFileFromTar(tr *tar.Reader, file string) ([]byte, error) {
 func HdrToTimespec(hdr *tar.Header) []syscall.Timespec {
 	return []syscall.Timespec{fileutil.TimeToTimespec(hdr.AccessTime), fileutil.TimeToTimespec(hdr.ModTime)}
 }
+
+func forbiddenXattr(xattr string) bool {
```

Comment:
As we are doing whitelisting, this function name (and logic) is a bit misleading. Can we turn it into `isXattrAllowed()`?